core/src/test/java/io/aiven/klaw/controller/AclControllerTest.java

```diff
@@ -224,20 +176,14 @@ public void getAcls1() throws Exception {
 
     when(aclControllerService.getAcls("testtopic")).thenReturn(topicOverview);
 
-    String res =
-        mvcAcls
-            .perform(
-                MockMvcRequestBuilders.get("/getAcls")
-                    .param("topicnamesearch", "testtopic")
-                    .contentType(MediaType.APPLICATION_JSON)
-                    .accept(MediaType.APPLICATION_JSON))
-            .andExpect(status().isOk())
-            .andReturn()
-            .getResponse()
-            .getContentAsString();
-
-    TopicOverview response = OBJECT_MAPPER.readValue(res, TopicOverview.class);
-    assertThat(response.getAclInfoList()).hasSize(1);
+    mvcAcls
+        .perform(
+            MockMvcRequestBuilders.get("/getAcls")
+                .param("topicnamesearch", "testtopic")
+                .contentType(MediaType.APPLICATION_JSON)
+                .accept(MediaType.APPLICATION_JSON))
+        .andExpect(status().isOk())
+        .andExpect(jsonPath("$.topicInfoList[*]", hasSize(1)));
```

Comment:
should this be aclInfoList ?

Reply:
@sekaiser can you recheck this if it should be aclInfoList ?

Reply:
it seems, you receive both (+ more):
```json{
"topicInfoList":[{"topicid":null,"sequence":null,"totalNoPages":null,"currentPage":null,"allPageNos":null,"topicName":null,"noOfPartitions":null,"description":null,"documentation":null,"noOfReplcias":null,"teamname":"Seahorses","cluster":null,"clusterId":null,"environmentsList":null,"showEditTopic":false,"showDeleteTopic":false,"topicDeletable":false}],

"aclInfoList":[{"sequence":null,"req_no":null,"acl_ip":null,"acl_ssl":null,"topicname":null,"topictype":"Producer","consumergroup":null,"environment":null,"environmentName":null,"teamname":"Seahorses","teamid":0,"operation":null,"permission":null,"transactionalId":null,"aclPatternType":null,"totalNoPages":null,"allPageNos":null,"possibleTeams":null,"currentPage":null,"showDeleteAcl":false}],

"prefixedAclInfoList":null,
"transactionalAclInfoList":null,
"topicHistoryList":null,
"promotionDetails":null,
"topicExists":false,
"schemaDetails":null,
"schemaExists":false,
"prefixAclsExists":false,
"txnAclsExists":false,
"topicDocumentation":null,
"topicIdForDocumentation":null
}
```